dataplane/linux/xdp_state.go

```diff
@@ -532,34 +533,35 @@ func (s *xdpIPState) newXDPResyncState(common *xdpStateCommon, isSource ipsetsSo
 	}, nil
 }
 
-func isValidMode(mode bpf.XDPMode, common *xdpStateCommon) bool {
-	for _, xdpMode := range common.xdpModes {
+func isValidMode(mode bpf.XDPMode, xdpModes []bpf.XDPMode) bool {
+	for _, xdpMode := range xdpModes {
 		if xdpMode == mode {
 			return true
 		}
 	}
 	return false
 }
 
-func (s *xdpIPState) getIPSetMembers(setID string, isSource ipsetsSource) (set.Set, error) {
-	return getIPSetMembers(s.ipsetIDsToMembers, setID, isSource)
+func (s *xdpIPState) getIPSetMembers(setID string, ipsSource ipsetsSource) (set.Set, error) {
+	return getIPSetMembers(s.ipsetIDsToMembers, setID, ipsSource)
 }
 
-// tryResync performs the resynchronization of the XDP state. It
-// modifies the BPF actions based on the state of XDP on the system
+// tryResync reconciles the system's XDP state (derived from xdpStateCommon)
```

Comment:
@fasaxc  deleted my comment overhaul . ..... but   is it ok  to keep this   tiny modification to remove resync and replace w/ reconcile but, if not ill remove this comment entirely >?

Reply:
updated..

Reply:
Still makes it sound like it's reconciling against the IP sets source, when it's really reconciling against the `xdpIPState`, using the IP sets source as a reference.